tests/functional/spec/features/supplementary_data/supplementary_data.spec.js

```diff
@@ -46,8 +46,13 @@ describe("Using supplementary data", () => {
       responseId,
     });
   });
+  it("Given I launch a survey using supplementary data, When I begin the interstitial block, Then I see the repeating supplementary data outside of the repeat piped in", async () => {
```

Comment:
Applies to many tests but I think we should make our Given/When/Then's generic, so wouldn't say on the interstitial block. Maybe something like "When I am outside a repeating section, Then I am able to see the list of items relating to a given supplementary data list item on the page"